backend/prisma/schema.prisma

```diff
@@ -0,0 +1,43 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model User {
+  id String @id @db.Uuid
+
+  imageUrl String
+  name     String
+  handle   String
+
+  followers  User[] @relation("UserToUser")
+  followed   User[] @relation("UserToUser")
+  posts      Post[] @relation("myPosts")
+  likedPosts Post[]
+}
+
+model Post {
+  id Int @id @default(autoincrement())
+
+  user   User   @relation("myPosts", fields: [userId], references: [id])
+  userId String @db.Uuid
+
+  timestampCreated  Int
```

Comment:
Could take advantage of the dateTime type, and also prisma's @createdAt and @updatedAt so it's handled for us

Reply:
Good point, I've just made a habit of avoiding dates since they're a bit more annoying to do arithmetic on in the database, but we don't need to do that here.